### ISSUE TITLE
model.eval for temperature scaling?

### ISSUE THREAD

**Issue by vivkul** [gpleiss/temperature_scaling#15]
Hi, I was wondering that as dropout is used, shouldn't we be using original_model.eval() for temperature scaling? As the logits obtained using one forward pass is used for multiple steps (instead of one forward pass for each step) to use optim.LBFGS. Thanks!

**Comment by pvmilk** [comment]
Wondering the same thing. The code might not use dropout as it were set to 0, but it obviously use batchnorm. 

Any answer on this yet?
